importer/importer.js

```diff
@@ -0,0 +1,72 @@
+'use strict'
+import fs from 'fs'
+import csv from 'csv'
+import EventEmitter from 'events'
+import {DirWatcher} from '../dirwatcher/dirwatcher.js';
+let filterCVS = (d) => {
+    let arr = d.toString().split('\n');
+    let data = [];
+    let [id, name, brand, company, price, isbn] = arr[0].split(',');
+    arr.forEach((a) => {
+        let line = a.split(',');
+        data.push(
+            {
+                [id]: line[0],
+                [name]: line[1],
+                [brand]: line[2],
+                [company]: line[3],
+                [price]: line[4],
+                [isbn]: line[5]
+            }
+        )
+    })
+    return data
+}
```

Comment:
this fn returns first line (wich holds collumn titles) as value, please fix that. It would be good if you expect random ammount of collumns instead of exactly 6. Also last line is empty line and shouldnt be a value.